packages/contracts/index.js

```diff
@@ -0,0 +1,7 @@
+const IExtendedResolver = require('./artifacts/contracts/IExtendedResolver.sol/IExtendedResolver.json')
+const OffchainResolver = require('./artifacts/contracts/OffchainResolver.sol/OffchainResolver.json')
+
+module.exports = {
+  IExtendedResolver,
+  OffchainResolver
+}
```

Comment:
What is this needed for?

Reply:
This should be a typescript file, then, and maybe should pull in the typechain artefacts instead.

Reply:
Do you have any pointer/reference to how to do that?

Reply:
Looks like I was mistaken, and we don't have typechain set up in this repo right now. Let's just convert this to TS.

Reply:
`client` and `gateway` is `typescript` project but this `contract` part is not ts so all written in js. Isn't it strange to only convert this file to TS, or are you requesting me to convert the entire package to ts?

Reply:
You're right, my mistake. We should open an issue to add typechain later, though.